.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+- package-ecosystem: composer
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  ignore:
+  - dependency-name: phpunit/phpunit
+    versions:
+    - "> 6.0"
```

Comment:
`phpunit/phpunit` was ignored in https://github.com/cakephp/authorization/pull/90#issuecomment-496783282.

It is now unnecessary as it is already `"phpunit/phpunit": "^8.5 || ^9.3"`